### ISSUE TITLE
where is lr_schedule? where u set  Polynomial 

### ISSUE THREAD

**Issue by oujieww** [meetshah1995/pytorch-semseg#107]
I have read many issues for this repo, I have see u ask us to off the lr_schedule and do not use Polynomial !!!but i can not find this setting in train.py!!!
  can u help me ? thank you.

and is the val dataset is right? benchmark of aug voc 2012 /train.txt include some image of val.txt of VOC 2012.
  thank you, hope some can help me

**Comment by raj-shah** [comment]
To use polynomial-decay learning rate, use the provided `poly_lr_scheduler` function in `misc.py`. To set it up in your training routine, modify `train.py` for something like: 

` iter = len(trainloader) * epoch + i`
`poly_lr_scheduler(optimizer, args.l_rate, iter)`

**Comment by oujieww** [comment]
@raj-shah  I will try it ,thank you so much  XD!
